src/index.js

```diff
@@ -63,12 +77,25 @@ const pluckIncludes = (tokens) => {
 
 const resolveNamespaceOrComponent = (namespaces, template) => {
   let resolveTemplate = template
+  const isNamespace = template.includes(":")
+
   // Support for SDC.
-  if (template.includes(":")) {
+  if (isNamespace) {
     const [namespace, component] = template.split(":")
     resolveTemplate = `@${namespace}/${component}/${component}`
   }
-  return Twig.path.expandNamespace(namespaces, resolveTemplate)
+  let expandedPath = Twig.path.expandNamespace(namespaces, resolveTemplate)
+
+  // If file not found and we are in namespace -> search deeper.
+  if (!existsSync(expandedPath) && isNamespace) {
+    const [namespace, component] = template.split(":")
+    let foundFile = findInChildDirectories(namespaces[namespace], component)
+    if (existsSync(foundFile)) {
```

Comment:
`findInChildDirectories` already does `existsSync` do we need to do it again here? is a `!== null` check enough at this point?

Reply:
yeah, i've removed it in function.